velox/py/plan_builder/PyPlanBuilder.h

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "velox/core/PlanNode.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/py/type/PyType.h"
+
+namespace facebook::velox::py {
+
+/// Called from when the Python module is loaded/initialized to register all
+/// resources and structures needed by Velox.
```

Comment:
Would it be possible to elaborate a bit about what resources and structures Velox needs?